docs/data/material/getting-started/templates/sign-in/SignIn.tsx

```diff
@@ -134,21 +163,8 @@ export default function SignIn() {
   return (
     <ThemeProvider theme={showCustomTheme ? SignInTheme : defaultTheme}>
       <CssBaseline />
-      <Stack
-        direction="column"
-        justifyContent="space-between"
-        sx={(theme) => ({
-          backgroundImage:
-            theme.palette.mode === 'light'
-              ? 'radial-gradient(ellipse at 50% 50%, hsl(210, 100%, 97%), hsl(0, 0%, 100%))'
-              : 'radial-gradient(at 50% 50%, hsla(210, 100%, 16%, 0.3), hsl(220, 30%, 5%))',
-          backgroundRepeat: 'no-repeat',
-          height: { xs: 'auto', sm: '100dvh' },
-          pb: { xs: 12, sm: 0 },
-        })}
-        component="main"
-      >
-        <Stack
+      <Stack direction="column" justifyContent="space-between">
+        <MuiStack
```

Comment:
```suggestion
        <Stack
```

Or another name, because you're also using Stack instances that weren't customized via `styled`

Reply:
I used `MuiStack` for instances that weren't customized. Should I use another one?

Reply:
Yeah, another name would be better. Otherwise, we don't know whether Stack is going through the styled function. I'd go with the same approach as above, some meaningful/contextual name.